### DISCUSSION TITLE
为啥使用 MyListItem 进行联网帮助页面请求会导致卡死

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#1405] [提问] [answered]
![image](https://user-images.githubusercontent.com/109946426/222961543-a4b447d6-5bd9-4f2d-9f65-dad1a99a64f2.png)

**Comment by LTCatt** [comment, marked as answer]
因为 MyListItem 需要读取页面的 Logo、标题、描述等来完成界面初始化，如果这时候需要联网获取，就会在联网部分结束前一直无法初始化，导致 UI 假死（
